test/UnitTests/MSTest.Core.Unit.Tests/Assertions/AssertTests.IsInstanceOfTypeTests.cs

```diff
@@ -3,107 +3,91 @@
 
 namespace Microsoft.VisualStudio.TestPlatform.TestFramework.UnitTests;
 
-extern alias FrameworkV1;
-extern alias FrameworkV2;
-
 using System;
 using System.Globalization;
 using System.Threading.Tasks;
-using MSTestAdapter.TestUtilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-using Assert = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
-using StringAssert = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.StringAssert;
-using TestClass = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute;
-using TestFrameworkV2 = FrameworkV2.Microsoft.VisualStudio.TestTools.UnitTesting;
-using TestMethod = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute;
+using global::TestFramework.ForTestingMSTest;
 
 public partial class AssertTests
 {
-    [TestMethod]
     public void IsInstanceOfType_WhenValueIsNull_Fails()
     {
-        static void action() => TestFrameworkV2.Assert.IsInstanceOfType(null, typeof(AssertTests));
-        ActionUtility.ActionShouldThrowExceptionOfType(action, typeof(TestFrameworkV2.AssertFailedException));
+        static void action() => Assert.IsInstanceOfType(null, typeof(AssertTests));
+        var ex = VerifyThrows(action);
+        Verify(ex.GetType() == typeof(AssertFailedException));
```

Comment:
You can use new style and do `ex is AssertFailedException`. Please apply this everywhere :)